charts/kube-prometheus-stack/templates/grafana/dashboards-1.14/alertmanager-overview.yaml

```diff
@@ -8,8 +8,8 @@ https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-promet
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  namespace: {{ template "kube-prometheus-stack.namespace" . }}
-  name: {{ printf "%s-%s" (include "kube-prometheus-stack.fullname" $) "alertmanager-overview" | trunc 63 | trimSuffix "-" }}
+  namespace: {{ template "grafana.namespace" . }}
+  name: {{ printf "%s-%s" (include "grafana.fullname" $) "alertmanager-overview" | trunc 63 | trimSuffix "-" }}
```

Comment:
Did you intend to change the name of the configmap as well here?

Reply:
@vsliouniaev No, sorry. Fixed.